<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ORIXVSBLT0067" ignored="0" name="Default test" tests="5" failures="5" timestamp="2022-12-05T13:48:42 IST" time="9.57" errors="0">
  <testcase name="runScenario" time="0.374" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at parallel.Login.user_enters_the_website_through_url_https_cloudstudio_visionet_com_login(Login.java:58)
at ✽.User enters the website through URL https://cloudstudio.visionet.com/Login(file:///C:/OLD-2-NEW/API/API_Training_B51/API_automation_training/CloudApp3/CloudApp3/src/test/resources/parallel/Forgot_Pwd.feature:4)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.035" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at parallel.Login.user_enters_the_website_through_url_https_cloudstudio_visionet_com_login(Login.java:58)
at ✽.User enters the website through URL https://cloudstudio.visionet.com/Login(file:///C:/OLD-2-NEW/API/API_Training_B51/API_automation_training/CloudApp3/CloudApp3/src/test/resources/parallel/Login.feature:4)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.022" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at parallel.Login.user_enters_the_website_through_url_https_cloudstudio_visionet_com_login(Login.java:58)
at ✽.User enters the website through URL https://cloudstudio.visionet.com/Login(file:///C:/OLD-2-NEW/API/API_Training_B51/API_automation_training/CloudApp3/CloudApp3/src/test/resources/parallel/Login.feature:4)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.038" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at parallel.Services.user_clicks_on_service_operations_from_the_navigation_menu(Services.java:44)
at ✽.User clicks on Service operations from the navigation menu(file:///C:/OLD-2-NEW/API/API_Training_B51/API_automation_training/CloudApp3/CloudApp3/src/test/resources/parallel/Serviceoperation.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="5.017" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at parallel.Services.user_navigates_to_account_section(Services.java:151)
at ✽.User Click on Logout button(file:///C:/OLD-2-NEW/API/API_Training_B51/API_automation_training/CloudApp3/CloudApp3/src/test/resources/parallel/Serviceoperation.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
